controllers/getArticleById.controllers.js

```diff
@@ -26,6 +26,13 @@ exports.updateItemById = async (req, res, next) => {
     const { article_id } = req.params;
     const { inc_votes } = req.body;
 
+    if (inc_votes === undefined) {
+      next({
+        status: 400,
+        msg: 'There was a problem with the input name',
+      });
+    }
+
     const updatedData = await updatedItem(article_id, inc_votes);
 
     if (updatedData === undefined) {
```

Comment:
This looks good John! But to avoid any future issues, I would make use of more `if/else if/else` statements - because when you invoke `next()` it doesn't exit the function, meaning even if `inc_votes` is `undefined` it might eventually try to call `updatedItem` and then do your `res.status().send()` even though there should only be an error.

Reply:
Done